pdr_backend/analytics/predictoor_dashboard/dash_components/callbacks.py

```diff
@@ -1,58 +1,43 @@
-from dash import Input, Output, State
+import copy
+
 import dash
-from pdr_backend.analytics.predictoor_dashboard.dash_components.util import (
-    get_feeds_data_from_db,
-    get_payouts_from_db,
-    get_user_payouts_stats_from_db,
-    filter_objects_by_field,
-    get_start_date_from_period,
-    get_date_period_text,
-    get_feed_ids_based_on_predictoors_from_db,
-    get_predictoors_data_from_payouts,
-)
-from pdr_backend.analytics.predictoor_dashboard.dash_components.view_elements import (
-    get_graph,
+from dash import Input, Output, State
+
+from pdr_backend.analytics.predictoor_dashboard.dash_components.app_constants import (
+    PREDICTOOR_TABLE_COLUMNS,
+    PREDICTOOR_TABLE_HIDDEN_COLUMNS,
 )
 from pdr_backend.analytics.predictoor_dashboard.dash_components.plots import (
     get_figures_and_metrics,
 )
 from pdr_backend.analytics.predictoor_dashboard.dash_components.util import (
+    filter_objects_by_field,
+    get_date_period_text,
+    get_feed_ids_based_on_predictoors_from_db,
+    get_payouts_from_db,
+    get_start_date_from_period,
     select_or_clear_all_by_table,
 )
-from pdr_backend.analytics.predictoor_dashboard.dash_components.app_constants import (
-    PREDICTOOR_TABLE_COLUMNS,
-    PREDICTOOR_TABLE_HIDDEN_COLUMNS,
+from pdr_backend.analytics.predictoor_dashboard.dash_components.view_elements import (
+    get_graph,
 )
 from pdr_backend.cli.arg_feeds import ArgFeeds
 
 
 # pylint: disable=too-many-statements
 def get_callbacks(app):
     @app.callback(
-        Output("feeds-data", "data"),
-        Output("predictoors-data", "data"),
-        Output("error-message", "children"),
-        Output("show-favourite-addresses", "value"),
-        Output("is-loading", "value"),
-        Input("data-folder", "data"),
+        [
+            Output("error-message", "children"),
```

Comment:
Looks like this error message output is alway going to be `None` now, so let's remove it.
Following up on this, what happens if there is no data in the lake, or something wrong with fetching the data, is the error displayed somewhere?

Reply:
I can display the error on command startup. I think that's in line with what we have already in CLI

Reply:
Maybe you can set the children of the error-message component before the app starts with the error if there is any